### PR TITLE
Add note about import of non-linear tiled images (unresolved)

### PR DESCRIPTION
**Pull request by lakshmih**

### CODE REVIEW

File: ext/cl_khr_external_memory.asciidoc [hunk 1]
```diff
@@ -586,3 +586,11 @@ while (true) {
...
 ----
 --
+
+
+=== Issues
+
+. How should the import of images that are created in external APIs with non-linear tiling be robustly handled?
++
+--
+*UNRESOLVED*
```
Comment (bashbaug) [inline ext/cl_khr_external_memory.asciidoc:596]:
```suggestion
*UNRESOLVED*
--
```

Can you please add the closing block to this issue?  Without this fix I'm getting the error:

> asciidoctor: WARNING: OpenCL_Ext.txt: line 111: unterminated open block

**Review by bashbaug — CHANGES_REQUESTED** [review]
Found one minor asciidoc bug.

Let's discuss the actual wording of the issue in tomorrow's teleconference.

**Comment by lakshmih** [comment]
Thanks Ben, I think I fixed it.

**Review by bashbaug — COMMENTED** [review]
Thanks, the asciidoctor build error is fixed.

Still need to approve the issue wording before merging, though.

**Review by bashbaug — APPROVED** [review]
Merging as discussed in the January 31st teleconference.

**Pull request merged**